packages/@aws-cdk/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -135,7 +72,7 @@ export interface HttpApiProps {
    *
    * @default - no default domain mapping configured. meaningless if `createDefaultStage` is `false`.
    */
-  readonly defaultDomainMapping?: DefaultDomainMappingOptions;
+  readonly defaultDomainMapping?: DomainMappingOptions;
```

Comment:
I think I understand the change here. 

To clarify I understood this correctly, `DomainMappingOptions` had a `stage` property that was unnecessary because its only usage was within `HttpStageOptions`. Is that correct?

Reply:
Yes that is correct.